Makefile

```diff
@@ -145,7 +145,7 @@ vet: ## Run go vet against code.
 	go vet -composites=false ./...
 
 ## Run tests.
-test test-v: TEST_VARS = KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" RECONCILE_RESYNC_PERIOD=5s
+test test-v: TEST_VARS = LOCALBIN=$(LOCALBIN) KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" RECONCILE_RESYNC_PERIOD=5s
```

Comment:
With the HELM var I think this def of localbin is no longer necessary

Reply:
Yeah I'll remove it and verify. Thanks!